tests/e2e-instrumentation-test.sh

```diff
@@ -86,7 +86,7 @@ for TEST in $TESTS; do
         --cache mem-cache-size=512 \
         --output format:json \
         --output option:parse-arguments \
-        --filter comm=echo,mv \
+        --filter comm=echo,mv,bpftrace \
```

Comment:
I guess you're doing the test with 'tracee' and not 'bpftrace', will remove this in the rebase, check if tests are good etc.